### DISCUSSION TITLE
Script Creating Empty Docker Images

### DISCUSSION THREAD

**Discussion by phiko73** [meisnate12/Plex-Meta-Manager#210] [Q&A]
Hello!

I am running the script in Unraid as shown for docker execution, but it seems that every time the script is run, it creates a new docker container with a strange name. (Ex: busy_zhukovsky, charming_antonelli, frosty_bouman, etc). 

Each of these containers are dead containers and don't do anything, but they fill up my docker image on Unraid, and since I run the script daily on a schedule, it is creating a new container for each run. So there are 7 empty containers per week.

Any thoughts on why this is happening?

**Comment by meisnate12** [comment]
seems like your not naming the container in the launch command those weird names are default container names. Im inexperienced with Unraid but how are you running the script? are you using the `-r` option? If youre not using the `-r` option you should be able to just leave the script running and it will kick off once a day at 3am

**Comment by xFlawless11x** [comment]
> Hello!
> 
> I am running the script in Unraid as shown for docker execution, but it seems that every time the script is run, it creates a new docker container with a strange name. (Ex: busy_zhukovsky, charming_antonelli, frosty_bouman, etc).
> 
> Each of these containers are dead containers and don't do anything, but they fill up my docker image on Unraid, and since I run the script daily on a schedule, it is creating a new container for each run. So there are 7 empty containers per week.
> 
> Any thoughts on why this is happening?

I'm running on unraid and do not have this issue, can you share your config screen?
